cEP-0019.md

```diff
@@ -0,0 +1,208 @@
+Meta-review System
+===========================
+
+|Metadata|                                   |
+|--------|-----------------------------------|
+|cEP     |0019                               |
+|Version |1.0                                |
+|Title   |Meta-review System                 |
+|Authors |Boxuan Li <liboxuan@connect.hku.hk>|
+|Status  |Proposed                           |
+|Type    |Process                            |
+
+Abstract
+--------
+
+This cEP describes the details of the process of implementing the meta-review
+system as a part of the
+[GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).
+
+Background
+----------
+
+People including the author of pull request respond to comments by attaching
```

Comment:
Keep this first sentence (and probably first paragraph) factually describing the feature we are going to use.  It is called `reactions` in GitHub.
Dont mentioned meta-review in the first sentence, which is a way of *using* reactions.

It is also worth describing what are the types of comments which might appear on a PR, and what is meant by "review comments".

Note there is a technical limitation in that reactions can not be placed onto the "review summary", which means we may need to describe what types of feedback should be put into the "review summary" text box, as it will be not able to be meta-reviewed.